test/Users.js

```diff
@@ -95,3 +97,169 @@ describe("2360: Users settings", () => {
       });
   });
 });
+
+describe("2370: Change password", () => {
+  before(async () => {
+    const loginResponseIngestor = await utils.getIdAndToken(appUrl, {
+      username: "adminIngestor",
+      password: TestData.Accounts["adminIngestor"]["password"],
+    });
+    accessTokenAdminIngestor = loginResponseIngestor.token;
+
+    const loginResponseUser1 = await utils.getIdAndToken(appUrl, {
+      username: TestData.Accounts["user1"]["username"],
+      password: TestData.Accounts["user1"]["password"],
+    });
+    userIdUser1 = loginResponseUser1.userId;
+    accessTokenUser1 = loginResponseUser1.token;
+
+    const loginResponseUser2 = await utils.getIdAndToken(appUrl, {
+      username: TestData.Accounts["user2"]["username"],
+      password: TestData.Accounts["user2"]["password"],
+    });
+    userIdUser2 = loginResponseUser2.userId;
+    accessTokenUser2 = loginResponseUser2.token;
+
+    db.collection("User").updateOne(
+      { username: TestData.Accounts["user2"]["username"] },
+      { $set: { authStrategy: "oidc" } },
+    );
+  });
+
+  after(async () => {
+    db.collection("User").updateOne(
+      { username: TestData.Accounts["user2"]["username"] },
+      { $set: { authStrategy: "local" } },
+    );
+  });
+
+  it("0010: should fail to change password with incorrect password", async () => {
+    return request(appUrl)
+      .post("/api/v3/users/password")
+      .send({
+        currentPassword: "wrongOldPassword",
+        newPassword: TestData.Accounts["user1"]["password"],
+        confirmPassword: TestData.Accounts["user1"]["password"],
+      })
+      .set("Accept", "application/json")
+      .set({ Authorization: `Bearer ${accessTokenUser1}` })
+      .expect(TestData.BadRequestStatusCode)
+      .then((res) => {
+        res.body.should.have.property(
+          "message",
+          "Current password is incorrect",
+        );
+      });
+  });
+
+  it("0020: should fail when new password and confirmation password do not match", async () => {
+    return request(appUrl)
+      .post("/api/v3/users/password")
+      .send({
+        currentPassword: "wrongOldPassword",
+        newPassword: TestData.Accounts["user1"]["password"],
+        confirmPassword: "wrongConfirmPassword",
```

Comment:
We should provide the correct password in the `currentPassword`, otherwise you do not know if it fails because of the wrong old password or the fact that the new password does not match